Lib/http/server.py

```diff
@@ -131,6 +132,7 @@
 """
 
 DEFAULT_ERROR_CONTENT_TYPE = "text/html;charset=utf-8"
+RANGE_REGEX_PATTERN = re.compile(r'bytes=(\d*)-(\d*)$', re.IGNORECASE)
```

Comment:
Strictly speaking, `\d` is *any* Unicode decimal digit which is weaker than `0...9`. So, either we force `re.ASCII` to restrict to ASCII parsing (and I think it's sufficient) or we replace `\d` by `[0-9]` (see https://datatracker.ietf.org/doc/html/rfc7233#appendix-C for what a DIGIT is)

Reply:
I thought this wouldn’t matter because the header fields are decoded from Latin-1 (in _http.client_ function __parse_header_lines_), which doesn’t have any non-ASCII digits. But I guess it would be clearer.